os_builders/galaxy.yml

```diff
@@ -0,0 +1,57 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: stfc_cloud
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: security
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
```

Comment:
Could we use 0.1.0 as this isn't release ready yet...

Reply:
done